### PR TITLE
HWDEF: correct default params file and readme for SPR H7

### PR DESCRIPTION
**Pull request by Hwurzburg**
found during prep of wiki page for this autopilot

### CODE REVIEW

**Comment by andyp1per** [comment]
@Hwurzburg the setup should match the documentation from here: http://seriouslypro.com/files/SPRacingH7EXTREME-Manual-latest.pdf - where did these changes come from? Do they match the documentation?

**Comment by Hwurzburg** [comment]
yes

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ The SPRacingH7 Extreme is a flight controller produced by [Seriously Pro Racing]
 
  - MCU - STM32H750 32-bit processor running at 400 MHz
  - 16MByte Serial NOR flash via QuadSPI
- - IMUs - 2x ICM20602
+ - 16MB Dataflash
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md:9]: This is not dataflash in the usual sense - you cannot put logs on it. If you want to change the name then call it "External Flash"

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ The SPRacingH7 Extreme is a flight controller produced by [Seriously Pro Racing]
...
+ - IMUs - 2x ICM20602 (only one used due to CPU limitations)
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md:10]: Only one enabled by default - it's perfectly possible for the user to enable both if they want

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md [hunk 3]
```diff
@@ -25,8 +26,8 @@ The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
 receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
 
  - SERIAL0 -> USB
- - SERIAL1 -> UART1 (DMA-enabled)
- - SERIAL2 -> UART2 (RCIN one wire, DMA-enabled)
+ - SERIAL1 -> UART1 (DMA-enabled) RCin
+ - SERIAL2 -> UART2 (SmartPort, DMA-enabled)
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md:30]: only onewire here, probably should make that clear

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md]: fixed

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md]: added

File: libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/README.md]: added

**Pull request merged**